Mathlib/Data/Complex/Order.lean

```diff
@@ -117,3 +117,35 @@ lemma neg_re_eq_abs {z : ℂ} : -z.re = abs z ↔ z ≤ 0 := by
 lemma re_eq_neg_abs {z : ℂ} : z.re = -abs z ↔ z ≤ 0 := by rw [← neg_eq_iff_eq_neg, neg_re_eq_abs]
 
 end Complex
+
+namespace Mathlib.Meta.Positivity
+open Lean Meta Qq Complex
+open scoped ComplexOrder
+
+private alias ⟨_, ofReal_pos⟩ := zero_lt_real
+private alias ⟨_, ofReal_nonneg⟩ := zero_le_real
+private alias ⟨_, ofReal_ne_zero_of_ne_zero⟩ := ofReal_ne_zero
+
+/-- Extension for the `positivity` tactic: `Complex.ofReal` is positive/nonnegative/nonzero if its
```

Comment:
```suggestion
/-- Extension for the `positivity` tactic: `Complex.ofReal'` is positive/nonnegative/nonzero if its
```

Reply:
This is a deliberate lie because all lemmas about `Complex.ofReal'` refer to it as `Complex.ofReal`. If you ask me, I think `Complex.ofReal` should be `Complex.ofRealHom` instead.